src/components/NcBreadcrumb/NcBreadcrumb.vue

```diff
@@ -171,6 +186,16 @@ export default {
 		}
 	},
 	computed: {
+		/**
+		 * TODO: drop on the 8.0.0 major, see title/name prop
+		 */
+		nameTitleFallback() {
+			if (this.name === null) {
```

Comment:
We need to be able to set an empty `name` for the `NcBreadcrumb` that shows the hidden breadcrumbs and only shows an `NcActions` dropdown without name.